src/node_dotenv.cc

```diff
@@ -261,12 +229,9 @@ Dotenv::ParseResult Dotenv::ParsePath(const std::string_view path) {
   return ParseResult::Valid;
 }
 
-void Dotenv::AssignNodeOptionsIfAvailable(std::string* node_options) const {
-  auto match = store_.find("NODE_OPTIONS");
-
-  if (match != store_.end()) {
-    *node_options = match->second;
-  }
+std::string Dotenv::GetNodeOptions() const {
```

Comment:
Nit: This function doesn't have to return a std::string, since the instance of this class will always live and has the lifespan of the node.js application. It can return std::string_view and avoid an unnecessary copy.

Reply:
AFAICT it has to be a `string` because `ParseNodeOptionsEnvVar` expects it to be one. I'm all for changing that... but maybe in a seperate PR?